CMakeLists.txt

```diff
@@ -90,8 +90,8 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
 option(ENABLE_KLEE_ASSERTS "Enable KLEE assertions" ON)
 if (ENABLE_KLEE_ASSERTS)
   message(STATUS "KLEE assertions enabled")
-  # We have to add the undefine to the flags, otherwise "-D-UDNDEBUG" will be added
-  list(APPEND KLEE_COMPONENT_CXX_FLAGS "-UDNDEBUG")
+  # We have to add the undefine to the flags, otherwise "-D-UNDEBUG" will be added
```

Comment:
```suggestion
  # We have to undefine the `NDEBUG`flag, which is the default added by CMake
```

Reply:
I was also at first puzzled by this comment, but it refers to why `-UNDEBUG` cannot be added to `KLEE_COMPONENT_CXX_DEFINES`, just like `-DNDEBUG` below.

Also, I think `NDEBUG` is defined by the compiler, but I will have to check again.